yarn.lock

```diff
@@ -2676,11 +2676,6 @@ eslint-module-utils@^2.6.0:
     debug "^2.6.9"
     pkg-dir "^2.0.0"
 
-eslint-plugin-eslint-plugin@^2.1.0:
```

Comment:
It's interesting why this dependency was here in the first place.
It got added in as standalone dependency in the lockfile with #125, but before that it was added in as a dependency for `eslint-plugin-react` with #117, although as far as I can tell it was always a dev dependency for `eslint-plugin-react`, so how it got in here in the first place is beyond me.